exasol_script_languages_developer_sandbox/templates/vm_bucket_cloudformation.jinja.yaml

```diff
@@ -1,15 +1,13 @@
 Resources:
-  {{bucket_name}}:
+  VMBucket:
```

Comment:
Is there a reason, why you changed to the hard coded name

Reply:
do, we have reason to keep it configurable, for example for testing or do we simply assume we test in a different account

Reply:
As discussed offline, we could deploy more stacks (assuming the stack names will be different), keeping the stack element names hard-coded, as AWS distinguishes between logical names and physical names.